dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/com/ibm/ws/cxf/transport/http/NoOpActivator.java

```diff
@@ -1,14 +1,5 @@
-/*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *******************************************************************************/
-package com.ibm.ws.jaxws23.cxf.rt.wsdl;
+package com.ibm.ws.cxf.transport.http;
```

Comment:
Why did you remove the copyright header?

Reply:
That was not intentional! Just a bad copy paste. Thanks for catching it!